pkg/assembler/graphql/examples/has_source_at.gql

```diff
@@ -53,8 +53,7 @@ query HasSourceAtQ2 {
   }
 }
 
-# this will return nothing as we are not ingesting package openssl in testdata
-query HasSourceAtQ3 {
+query Q3 {
```

Comment:
Let's give this a unique name (like `SourceAtQ3`), @lumjjb was making the other examples be unique as these files are now load bearing for the JS work

Reply:
oh didn't realize that name had changed. Accidental overwrite. Fixed.